tests/ctags/simple.tcl

```diff
@@ -1,10 +1,55 @@
-# proc comment
-proc simple1
+oo::class create X {
+    superclass Y;
 
-class tcl_class
+    method add v {
+    }; method doSomething {v} {
+    }
+}
 
-itcl::class itcl_class
+class create ShouldNotBeCaptured {
+}
 
-public method method1
-protected method method2
-private method method3
```

Comment:
Not sure if it matters but the new tests don't include public/private methods, TclOO supports these according to https://wiki.tcl-lang.org/page/TclOO+Tutorial, but I don't know if it ctags handles it at all?